packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -582,6 +583,24 @@ export class ProjectLifecycleManager {
 
       throw getError('CONFIG_VALIDATION_ERROR', 'configFile', file || null, errMsg)
     })
+
+    return this.ctx._apis.configApi.validateLaunchpadConfigBreakingChanges(
+      config,
+      (type, obj) => {
+        const error = getError(type, obj)
+
+        this.ctx.onWarning(error)
+
+        return getError(type, obj)
```

Comment:
not sure this return is necessary since we created this func inline., however we can return `return error` since you created it on line 590.